src/smart-components/workspaces/WorkspaceListTable.tsx

```diff
@@ -150,14 +193,22 @@ const WorkspaceListTable = () => {
             </ResponsiveActions>
           }
         />
-        <DataViewTable
-          isTreeTable
-          aria-label="Workspaces list table"
-          ouiaId="workspaces-list"
-          columns={columns}
-          rows={rows}
-          bodyStates={{ loading: <SkeletonTableBody rowsCount={10} columnsCount={2} /> }}
-        />
+        {isLoading ? (
+          <SkeletonTable rowsCount={10} columns={columns} variant={TableVariant.compact} />
```

Comment:
I don't think there's a need to have `isLoading` part since you have loading states in the DataViewTable.